### ISSUE TITLE
WebEx 41.6.4.8 doesn't work with NVDA

### ISSUE THREAD

**Issue by geoffshang** [nvaccess/nvda#12570]
The latest WebEx release at time of writing completely fails to work with NVDA.  NVDA is unable to focus the window and even reports the window title and other items as being something else.

I realise this is probably an issue with WebEx, and I did report this issue to them.  But since it still seems to work with JFW and Narrator and NVDA provides all this lovely logging, I thought I'd better check in case it's something NVDA needs to deal with.

### Steps to reproduce:

please note: I already had WebEx installed and configured when the problem arose.

1.  Open Cisco WebEx or alt-tab to an already running copy.

2.  Try to do pretty much anything.

### Actual behavior:

App fails to gain focus.

Title reports as being another app (e.g. explorer).

Object navigator sees nothing.

NVDA + TAB says absolutely nothing.

### Expected behavior:

App should work as normal.  Normal for WebEx does unfortunately include a lot of random-sounding numbers.  I hacked up an app module to silence this but this issue occurs without it loaded.

#### NVDA installed/portable/running from source:

Both installed and run from installer.

#### NVDA version:

Installed version is 2019.2.1, run from installer is 2020.4.  The log was produced by 2020.4.

#### Windows version:

Version	10.0.19042 Build 19042

#### Name and version of other software in use when reproducing the issue:

Cisco WebEx 41.6.4.8

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

As noted, this happens with 2019.2.1 and 2020.4.  I have not tried any other version of NVDA.

JFW 2018 and current Narrator both seem to cope OK with this version of WebEx.

#### If add-ons are disabled, is your problem still occurring?

Yes.  I am running from the installer which means no add-ons.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes.  Note that I ran it from the installed copy which is 2019.2.1.  I need this old version due to an add-on I need for another app.

[nvda webex issue.txt](https://github.com/nvaccess/nvda/files/6688348/nvda.webex.issue.txt)

**Comment by geoffshang** [comment]
WebEx 41.6.5.10 was just released and the issue persists.

**Comment by geoffshang** [comment]
WebEx 41.6.6.14 resolves this issue.
